.travis.yml

```diff
@@ -42,5 +42,4 @@ before_install:
   - if [[ ! $GITHUB_TOKEN ]]; then echo "no github token"; fi
 
 after_script:
-  - if [[ $TRAVIS_PHP_VERSION = '5.6' ]]; then php vendor/bin/coveralls -v ; fi
```

Comment:
Are you sure we're using for doctrine/migrations? Do you want me to put it back?